packages/postcss-minify-selectors/src/index.js

```diff
@@ -233,7 +237,9 @@ function pluginCreator() {
             }
           }
         });
-        selectors.nodes.sort();
+        if (opts.preserveOrder !== true) {
```

Comment:
Please avoid using `no`/`preserve`/etc, mostly such names are very misleading, let's use `sort: boolean`, by default `true` and when it is `false` we will do nothing